video_xblock/static/html/student_view.html

```diff
@@ -17,4 +17,28 @@ <h3 class="hd hd-2">{{display_name}}</h3>
             <a href="{{ handout }}" class="download-handout" download="{{ handout_file_name }}">{% trans 'Download Handout' %}</a>
         </li>
     {% endif %}
+    {% if download_transcript_allowed and transcripts %}
+        <li class="video-transcript video-download-button-custom">
+            <a  class="download-transcript" id="download-transcript-link" href="#">{% trans 'Download transcript' %}</a>
+            <script>
+                var transcripts = {
+                {% for transcript in transcripts %}
+                 '{{transcript.lang}}': {
+                        'label': '{{transcript.label}}',
+                        'url': '{{transcript.url}}',
+                 },
+                {% endfor %}
+                };
+
+                function chooseTransciptForDownload(language){
+                    language = language || 'en';
+                    var currentTranscript = transcripts[language] || transcripts[Object.keys(transcripts)[0]]
```

Comment:
Are you sure `Object.keys(transcript)[0]` will return same element as `transcripts[0]` would before conversion into an object?